src/services/errors.js

```diff
@@ -1,4 +1,8 @@
 module.exports= {
+	ArticleDoesNotExistForCommentError : { 
+		name : 'ArticleDoesNotExistError', 
```

Comment:
the name should be updated as well. it being the same as the other error we just added will make one of them not return the right status code.